### PR TITLE
Homepage cover rework

### PR DESCRIPTION
**Pull request by chalin**
- Precursor to #789
- Preview: https://deploy-preview-794--opentelemetry.netlify.app/

/cc @tedsuo @amye

---

### Screenshots

Before:

> image

After:

> image

On mobile:

>

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -3,48 +3,35 @@ title: OpenTelemetry
...
+</a>
+
+<div class="h3 mt-2">Get started!</div>
+
+<div class="l-get-started-buttons">
+
+- [Collector]({{< relref "/docs/collector/getting-started" >}})
+- [Go]({{< relref "/docs/go/getting-started" >}})
+- [.Net]({{< relref "/docs/net/getting-started" >}})
+- [Javascript]({{< relref "/docs/js" >}})
```
Comment (chalin) [inline content/en/_index.md:23]: **Getting started** button order is based on analytics.

File: content/en/_index.md [hunk 2]
```diff
@@ -3,48 +3,35 @@ title: OpenTelemetry
 show_banner: true
 ---
 
-{{< blocks/cover title="" image_anchor="center" height="min" color="dark" >}}
-<div class="mx-auto">
-    <div class="col-sm-5 offset-sm-7 card bg-transparent border-0">
-      <img class="card-img-top mx-auto" src="/img/logos/opentelemetry-stacked-color.png">
-      <div class="card-body">
-        <h3 class="card-title text-left">An observability framework for cloud-native software.</h3>
-        <p class="card-text text-left"><small>OpenTelemetry is a collection of tools, APIs, and SDKs. You can use it to instrument, generate, collect, and export telemetry data (metrics, logs, and traces) for analysis in order to understand your software's performance and behavior.</small></p>
```
Comment (chalin) [inline content/en/_index.md:12]: This text has been moved into the lead block below.

File: content/en/_index.md (hunk 1 above)
Comment (austinlparker) [reply content/en/_index.md:23]: nit (and double check me here) but should it be `.NET` and `JavaScript`?

**Review by austinlparker — APPROVED** [review]
lgtm outside of comments

File: content/en/_index.md (hunk 1 above)
Comment (chalin) [reply content/en/_index.md:23]: Yes, and yes! Thanks for catching that.

Note that the title of https://opentelemetry.io/docs/js/ is "Javascript". I'll fix that later.

**Pull request merged**